src/Corax/IndexWriter.cs

```diff
@@ -56,11 +56,20 @@ public IndexWriter([NotNull] StorageEnvironment environment)
             _environment = environment;
             _transactionPersistentContext = new TransactionPersistentContext(true);
             Transaction = _environment.WriteTransaction(_transactionPersistentContext);
-
+            _ownsTransaction = true;
             _postingListContainerId = Transaction.OpenContainer(PostingListsSlice);
             _entriesContainerId = Transaction.OpenContainer(EntriesContainerSlice);
         }
         
+        // _transactionInjected disable transaction commit in instance of IndexWriter because it's done by Server. 
```

Comment:
Outdated comment? It tells about `_transactionInjected` which we don't have any longer. Do we really need that comment?

Reply:
Outdated. I missed that comment.